### PR TITLE
Cisco Spark

### PR DESCRIPTION
**Pull request by nshap**
Fixes https://github.com/demisto/server/issues/7779.

The implemented commands can be seen in the .yml file. It's straightforward with the API docs, in addition to two "human" commands: "send-message-to-person" and "send-message-to-room", which allow sending a message by name / email.

The playbook sends to the user: "ciscosparktestmail@gmail.com" and tests the following functionality:
* Create a team (also creates a room)
* Send messages to a room - via roomId / room name
* Send messages to a person - via personId / email
* List messages in a room
* Delete messages
* Delete a team / room

### CODE REVIEW

File: Integrations/integration-CiscoSpark.yml [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
...19WmUwVvt+vWoxd3TwAAADPQuFnDFnEzlUvkaf5LsterD71QOP+nTQWpt1acVxTPG8BqAmXV5d9DzD3pJ9ZfSbr14psriwYO7zPa1M4VAgDKWAMAvVbtO3aa/8L+V3fScC+T2nmtSOz98Ptf+rgPH8mdEgAAwIw9e2D5uVff2yA2FCSL+WcCi4OAma97i8xjY0TaUXoFPOos//rJF+ee9BebrqaKlbtyT5nqeUEAQBlzAKDXtJVb6xc//uG3HTTgn7BuCP28CtvOvfkOd0QAAAAz17xl09atO7To2qJd464de7u57z6/5vq+/1suVl+KFLNP+vyxtKBH8Vh2AoEHv/XPPjZBrLkUI45/+PztQaN6TWjaopEbAQBFAFB+ug0Y7P584YefsTLAE7z9//iG2HL5nevNW7dtxx0RAACgimnaonGrSQGjJR5+AySRs6avWXc4590X3pojnroYKeaemlr8lpsGv2HpV1l4+kqcWHs4+73JYR4xQ0cNHGfq5wEBAGUKAYDeQM9x47ddee86IcDj1cufFomQ2JQM7n4AAACo1qFL2+4XP3715qH3nhFb35gplp6X/TVUIPuYl0g9WlW7+3uImScmivzTM4qHT7z+0XPCY8JQX3M57gQAlKkEAHoTZoRJX/zkJ8FwgEed+O+GePm9L76vWbu2M3c7AAAAFLP4SzWLBo3rNV23N//0S++sEU9fSRR5p6aK3BOTRc7r40XW8bEi/egoXePYXJf0uzPGXz9HQv7pQPHCW3NFzsqYHfr9Urx3zAgBAGVKAUDDps2abzp2vmDftR9/169lT+P+4Wb9P/DpDTFdHRXHXQ4AAAD3ZWVlZd25R4e+Hbq37hui8U059Nbm71789xLx9OU4kXPcW6Qf8TDDxr9H8batuxIr1h3N+ihmpmRZxx5t+9Zv6NrYHI8xAQBlSgHAn1KXrd348mc3aeA/ROkn/guMjE20sbW1464GAACAh9apa/seC9dn7dr4St6beXsi31x/Ienm6ksaseBMkMg67lXceDbNif08xZwTU8TS81Kx+nzcL3N2qd5c/8rcNxNmqhab+zElAKBMMQBwqlXbeeWBY2de/OSmYE6ABy/7t+fdL39o0qJlK+5gAAAAeCJePh4Bu06s/b+X9cMELieIpeekxV3m9asKZB/3Ehm6hrXRzuSv+36zT/iIvFMBxd/7+X/l
```
Comment (meirwah) [inline Integrations/integration-CiscoSpark.yml]: add a description of expected format

Comment (meirwah) [inline Integrations/integration-CiscoSpark.yml]: add a description of expected format

Comment (meirwah) [inline Integrations/integration-CiscoSpark.yml]: add a description of expected format

File: Integrations/integration-CiscoSpark.yml [hunk 2]
```diff
@@ -0,0 +1,751 @@
+commonfields:
+  id: Cisco Spark
+  version: -1
+name: Cisco Spark
+display: Cisco Spark
+category: Messaging
```
Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: Are you sure this is the right category?

File: Integrations/integration-CiscoSpark.yml (hunk 1 above)
Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this part should describe how to get the token and not this description

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: trim / at the end of the server from param

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: I think we already have this in encodeURL which is commonly used

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this should be done on the server url to begin with before adding the v1

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: you can do this by using replaceInTemplatesAndRemove,
The url include url/%key% and args key=value, so after using it you will have url/value

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: seeme like a lot of outputs are missing

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this relates to the adding request arg

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this relates to the commands part in the yml

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this is for getting the server from params

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this is for buildArgs

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: this is for getting the server from params

Comment (Spivaka) [inline Integrations/integration-CiscoSpark.yml]: detailed description

**Pull request merged**